suite/tests/client-interface/drx-scattergather.c

```diff
@@ -176,7 +196,19 @@ test_avx2_gather(void (*test_func)(uint32_t *, uint32_t *, uint32_t *),
                  uint32_t *test_idx_vec, uint32_t *output_xmm_ymm OUT)
 {
     memset(output_xmm_ymm, 0, CONCAT_XMM_YMM_U32 * sizeof(uint32_t));
+#    ifdef UNIX
+    byte ymm_buf[32];
+    memset(ymm_buf, 0, sizeof(ymm_buf));
+    byte ref_buf[32];
+    memset(ref_buf, 0, sizeof(ref_buf));
```

Comment:
suggest: change all of these declare;memset to declare = {0} for single-line init.

Reply:
Done.